dawn/prototype/AtlasCartesianWrapper.cpp

```diff
@@ -0,0 +1,100 @@
+//===--------------------------------------------------------------------------------*- C++ -*-===//
+//                         _       _
+//                        | |     | |
+//                    __ _| |_ ___| | __ _ _ __   __ _
+//                   / _` | __/ __| |/ _` | '_ \ / _` |
+//                  | (_| | || (__| | (_| | | | | (_| |
+//                   \__, |\__\___|_|\__,_|_| |_|\__, | - GridTools Clang DSL
+//                    __/ |                       __/ |
+//                   |___/                       |___/
+//
+//
+//  This file is distributed under the MIT License (MIT).
+//  See LICENSE.txt for details.
+//
+//===------------------------------------------------------------------------------------------===//
+
+#include "AtlasCartesianWrapper.h"
+
+Point AtlasToCartesian::cellMidpoint(const atlas::Mesh& mesh, int cellIdx) const {
+  const atlas::mesh::HybridElements::Connectivity& cellNodeConnectivity =
+      mesh.cells().node_connectivity();
+
+  const int missingVal = cellNodeConnectivity.missing_value();
```

Comment:
Sorry for the probably stupid question: what is the missing value here? I was looking through the atlas code and couldn't figure it out ... it's just a curiosity